modules/op-starting-pipelines.adoc

```diff
@@ -59,10 +61,10 @@ To see Pipelines created using the *Pipeline Builder* in the *Topology* view, cu
 ====
 +
 .. On the left navigation panel, click *Topology*, and click on the application to see the Pipeline Runs listed in the side panel.
-.. In the *Pipeline Runs* section, click *Start Last Run* to start a new Pipeline Run with the same parameters and resources as the previous ones. This option is disabled if a Pipeline Run has not been initiated.
+.. In the *Pipeline Runs* section, click *Start Last Run* to start a new pipeline run with the same parameters and resources as the previous ones. This option is disabled if a pipeline run has not been initiated.
```

Comment:
Should "as the previous ones" be "as the previous one"? My understanding is that **Start Last Run** only relates to the previous run. It is certainly worth checking and updating the sentence to be explicit about this.